jamovi/surv.a.yaml

```diff
@@ -10,23 +10,24 @@ options:
     - name: data
       type: Data
 
+    - name: elapsed
+      title: Time
```

Comment:
what do you think about 'Time Elapsed'? i have a slight preference for it, but don't mind if you prefer `Time`

Reply:
That's ok. I made the change.